core/src/main/java/dk/alexandra/fresco/framework/network/Network.java

```diff
@@ -63,4 +72,51 @@ default void sendToAll(byte[] data) {
     }
   }
 
+  /**
+   * Creates a map of party IDs to network address (uses localhost and supplied ports) information.
+   *
+   * @param n the number of parties
+   * @param ports the ports to be used
+   * @return a map of party IDs to network address information
+   */
+  static Map<Integer, NetworkConfiguration> getNetworkConfigurations(int n,
```

Comment:
These utility methods should not be added to the `Network` interface . If anything they should be added to some separate static Network Configuration utility class.

Reply:
Moved into a new class called NetworkUtil in the test package